metrics/web/test/dashboard/presentation/widgets/metrics_table_test.dart

```diff
@@ -115,8 +116,8 @@ void main() {
           ),
         );
 
-        expect(find.byType(ProjectMetricsTile), findsNothing);
-        expect(find.text(DashboardStrings.noConfiguredProjects), findsNothing);
+        expect(find.byType(NoSearchResultsPlaceholder), findsOneWidget);
+        expect(find.text(DashboardStrings.noSearchResults), findsOneWidget);
```

Comment:
It seems like we should not test that the `DashboardStrings.noSearchResults` text displayed since we've already tested it in the `NoSearchResultsPlaceholder ` tests.